src/__tests__/index.should-class-name-update.js

```diff
@@ -87,8 +89,8 @@ test.skip('shouldClassNameUpdate is specific to the component instance', () => {
 
         return (
           <div>
-            <Div color={count === 3 ? 'purple' : 'red'} />
-            <Div color={count === 3 ? 'green' : 'blue'} />
+            <Div css={{color: count === 3 ? 'purple' : 'red'}} />
+            <Div css={{color: count === 3 ? 'green' : 'blue'}} />
```

Comment:
These elements use `color` as a prop, but the prop is ignored because `propsAreCssOverrides` is not being used (I rendered this JSX when fixing the test, and thought there was a bug when the color never changed). Wrapping the old prop in a `css` object to make it "valid," so other newbies like myself do not get confused when reading the test.